implementation/leprechaun/src/shared/api/http-exceptions.ts

```diff
@@ -0,0 +1,62 @@
+import { HttpException, HttpStatus } from '@nestjs/common';
+import { ApiProperty } from '@nestjs/swagger';
+
+export const ErrorKind: Record<string, [string, number]> = {
+  SchemaMismatchException: ['schema-mismatch-exception', HttpStatus.UNPROCESSABLE_ENTITY],
+  NotFoundException: ['resource-not-found-exception', HttpStatus.NOT_FOUND],
+  InternalServerError: ['internal-server-error', HttpStatus.INTERNAL_SERVER_ERROR],
+} as const;
```

Comment:
**suggestion:** The type annotation discards `as const`. You probably wanted to do this:
```suggestion
export const ErrorKind = {
  SchemaMismatchException: ['schema-mismatch-exception', HttpStatus.UNPROCESSABLE_ENTITY],
  NotFoundException: ['resource-not-found-exception', HttpStatus.NOT_FOUND],
  InternalServerError: ['internal-server-error', HttpStatus.INTERNAL_SERVER_ERROR],
} as const satisfies Record<string, [string, number]>;
```